Libraries/Components/TextInput/TextInput.js

```diff
@@ -675,6 +674,11 @@ const TextInput = React.createClass({
       children = <Text>{children}</Text>;
     }
 
+    let selection = this.props.selection;
```

Comment:
Any reason why this is different from the way iOS handles selection? I'd prefer it to be the same.
Also, here `end` is `selection.end` which is null, while on iOS `end` is `selection.start`.